clang/lib/DPCT/Diagnostics.inc

```diff
@@ -233,5 +233,5 @@ DEF_WARNING(EXPERIMENTAL_FEATURE, 1108, "'%0' was migrated with the experimental
 DEF_COMMENT(EXPERIMENTAL_FEATURE, 1108, "'{0}' was migrated with the experimental feature {0} which may not be supported by all compilers or runtimes. You may need to adjust the code.")
 DEF_WARNING(DEVICE_UNSUPPORTED_CALL_FUNCTION, 1109, "%0 cannot be called in a SYCL kernel or by functions called by the kernel. You may need to adjust the code.")
 DEF_COMMENT(DEVICE_UNSUPPORTED_CALL_FUNCTION, 1109, "{0} cannot be called in a SYCL kernel or by functions called by the kernel. You may need to adjust the code.")
-DEF_WARNING(REGISTER_USAGE, 1110, "The total local variable size in device function exceeds 128 bytes and may cause high register pressure. Consider replacing sub-group size with smaller value to reduce register pressure for better performance.")
-DEF_COMMENT(REGISTER_USAGE, 1110, "The total local variable size in device function exceeds 128 bytes and may cause high register pressure. Consider replacing sub-group size with smaller value to reduce register pressure for better performance.")
+DEF_WARNING(REGISTER_USAGE, 1110, "The total declared local variable size in device function \"%0\" exceeds 128 bytes and may cause high register pressure. Consider replacing sub-group size with smaller value to reduce register pressure for better performance.")
+DEF_COMMENT(REGISTER_USAGE, 1110, "The total declared local variable size in device function \"{0}\" exceeds 128 bytes and may cause high register pressure. Consider replacing sub-group size with smaller value to reduce register pressure for better performance.")
```

Comment:
The total declared local variable size in device function \"{0}\" exceeds 128 bytes and may cause high register pressure. Consult with your hardware vendor to find the total register size available and adjust the code or use smaller sub-group size to avoid high register pressure.